README.md

```diff
@@ -411,6 +411,25 @@ change the UUID.
 You can't. This is because historical bookings still refer to that prison.
 Disable it instead (see above).
 
+#### Merging two prisons
+
+If two formerly-separate parts of an estate now have the same booking team and
```

Comment:
Yes, but it requires a bit of interpretation.  How about leading with an explanation of estate?  Maybe: 

"An `estate` is the way the application groups prisons that share common characteristics but otherwise handle booking separately."